src/Symfony/Component/HttpFoundation/Session/Storage/Handler/PdoSessionHandler.php

```diff
@@ -395,7 +395,7 @@ public function close()
             $this->gcCalled = false;
 
             // delete the session records that have expired
-            $sql = "DELETE FROM $this->table WHERE $this->lifetimeCol + $this->timeCol < :time";
+            $sql = "DELETE FROM $this->table WHERE $this->lifetimeCol < :time - $this->timeCol";
```

Comment:
isn't is the same?
`a + b < c` === `a < c - b`

Reply:
no, in postgres 
a and b - integer type (32 bit)
a+b > integer (2^32/2) - it's error
a-b <=  integer and accepted by postgres

Reply:
this problem in postgres only

Reply:
is it possible to use unsigned types? or larger types e.g. long (64 bits instead of 32)

Reply:
Creating tables is in the same package. But to increase the amount of data due to improper use is incorrect

Reply:
Moreover, the sessional table contains a lot of records, and the losses will be large